src/Jutul.jl

```diff
@@ -38,6 +37,13 @@ module Jutul
     using Crayons
     using Crayons.Box
 
+    # AD
+    import ForwardDiff
+
+    import DifferentiationInterface: AutoSparse, prepare_jacobian, jacobian
+    import SparseConnectivityTracer: TracerLocalSparsityDetector
```

Comment:
Any reason to use the local sparsity detector? What failed when you tried the global one?

Reply:
> Any reason to use the local sparsity detector? What failed when you tried the global one?

There was a searchsortedfirst call in a low-level interpolation routine that failed:

`ERROR: TypeError: non-boolean (SparseConnectivityTracer.GradientTracer{SparseConnectivityTracer.IndexSetGradientPattern{Int64, BitSet}}) used in boolean context`